backend/frontend/templates/frontend/page_course_component_details/page_project_details.html

```diff
@@ -0,0 +1,110 @@
+{% extends "frontend/page_course_component_details/base.html" %}
+{% load static %}
+
+{% block project_heading %}Project Details{% endblock %}
+{% block project_content %}
+    {% if project.submission_type_nice == "link"%}
+        <div class="grid gap-4 my-4">
+            <div  class="bold p-2 border-2 rounded-md grid gap-2">
+                <p class="bold p-2 border-2 rounded-md ">
+                    Title: {{ project.title }}
+                </p>
+            
+                <div class="bold p-2 border-2 rounded-md ">
+                    Tag names: 
+                    {% with project.tag_names|length as array_length %}
+                        {% for item in project.tag_names %}
+                            {{ item }}
+                            {% if not forloop.last %},{% endif %}
+                        {% endfor %}
+                    {% endwith %}
+                </div>
+            
+                <div class="bold p-2 border-2 rounded-md ">
+                    <p class="bold">
+                        Status : {{board_status}}
+                    </p>
+                </div>
+            </div>
+        
+            <div class="bold p-2 border-2 rounded-md grid gap-2">
+                {% if user.email != project.recruit_user_names|first %}
+                    <div class="bold p-2 border-2 rounded-md ">
+                        Assignees:
+                        <ul class="mx-8">
+                            <li class="bold">
+                            {{ project.recruit_user_names|first }}
+                            </li>
+                        </ul>
+                    </div>
+                {% endif %}
+        
+                <div class="bold p-2 border-2 rounded-md ">
+                    Reviewers: 
+                    <ul class="mx-8">
+                        {% for reviewer in project.reviewer_user_names%}
+                        <li class="bold">{{reviewer}}</li>
+                        {%endfor%}
+                    </ul>
+                </div>
+            </div>
+
+
+            
+            <div class="grid bold p-2 border-2 rounded-md gap-2">
+                <p class="bold p-2 border-2 rounded-md" >
+                    Start Date: {{project.start_time}}
+                </p>
+
+                <p class="bold p-2 border-2 rounded-md">
+                    Due Date: {{project.due_time}}
+                </p>
+            </div>
+
+            <div class="bold p-2 border-2 rounded-md ">
+                <p class="bold">
+                    Content URL: 
+                    <a class="{{styles.link}}" href="{{project.content_url}}">{{project.content_url}}</a>
+                </p>
+            </div>
+
+            <div class="grid bold p-2 border-2 rounded-md gap-2">
+                <h6>Link to your work</h6>
+                
+                <div>Current link submission: 
+                    <a class="{{styles.link}}" href="{{project.link_submission}}">
```

Comment:
@kingraphaii I've head this type of comment before from sheena, which was mainly about leveraging the tools that django provides, in that particular instance it was about using django built in forms rather than manually creating forms using html.

What is comment in reference to? is it about the link_submission mechanism?

Reply:
@kingraphaii oh I see, I understand. I thought it was a reference to another issue.